features/FEATURE_LWIP/lwip-interface/EthernetInterface.cpp

```diff
@@ -24,12 +24,35 @@ EthernetInterface::EthernetInterface()
 {
 }
 
+bool validate_str(const char *str, const int len)
```

Comment:
Yes, lwip validates the addresses. Added enforcing of null terminator and removed validate_str functions.